From 9dd6be04080762e56bfdae314b5f12df7189173f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 30 Jan 2022 17:29:49 +0200
Subject: [PATCH] [SW-0] gaudi: security: do not redefine sram_size

SRAM_SIZE is already defined in address_space.h included from specs.
No need to redefine it, so remove the SRAM_SIZE define.

Change-Id: Id61d8d73f624c6b76850454460eddc0781954f6a
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi/arm/security.c b/arch/gaudi/arm/security.c
index 239b4fe..48e06d8 100644
--- a/arch/gaudi/arm/security.c
+++ b/arch/gaudi/arm/security.c
@@ -1943,7 +1943,6 @@
 	preboot_puts("Security init finished.\n");
 }
 
-#define SRAM_SIZE	0x1400000	/* 20MB */
 #define SRAM_END	(SDRAM_BASE + SRAM_SIZE)
 
 enum preboot_sec_rr_names {
